PaymentSystem/Server/Application/Currencies/CurrencyManager.cs

```diff
@@ -3,22 +3,28 @@
 using System.Linq;
 using System.Threading.Tasks;
 
-namespace PaymentSystem.Server
+namespace PaymentSystem.Server.Application.Currencies
 {
-    public static class CurrencyManager
+    public class CurrencyManager : ICurrencyManager
     {
-        public static List<string> Currencies { get; }
+        private List<string> Currencies { get; }
 
-        static CurrencyManager()
+        public CurrencyManager()
         {
             Currencies = new List<string>
             {
                 "USD",
                 "EUR",
                 "GBP",
                 "MDL",
+                "BTC",
                 "EC"
             };
         }
+
+        public List<string> GetCurrencies()
```

Comment:
I think it would be nice to keep the Currencies as a property.
```csharp
public IEnumerable<string> Currencies { get; }
```
`IEnumerable` so we're not allowing others to modify the currencies.

Reply:
Interesting thought, this might reduce the chance for future problems, thanks!